index.bs

```diff
@@ -1043,23 +1159,18 @@ partial interface HTMLIFrameElement {
         :   [=PermissionsPolicyViolationReportBody/columnNumber=]
         ::  null
         :   [=PermissionsPolicyViolationReportBody/disposition=]
-        ::  "enforce"
+        ::  |disposition|
 
     1. If the user agent is currently executing script, and can extract the
       source file's URL, line number, and column number from |settings|, then
       set |body|'s [=PermissionsPolicyViolationReportBody/sourceFile=],
       [=PermissionsPolicyViolationReportBody/lineNumber=], and
       [=PermissionsPolicyViolationReportBody/columnNumber=] accordingly.
 
-    1. If |group| is omitted, set |group| to "default".
```

Comment:
Are we ever reacting to it being null here? Maybe I'm misunderstanding https://github.com/w3c/webappsec-permissions-policy/pull/529#discussion_r1357250611, but I expected to see a null check.

Reply:
My intention here is actually to put the null check into Reporting -- ideally, even with a null `endpoint`, this *would* generate a report for JavaScript reporting observers, but not actually send it out anywhere over the wire.

Currently Reporting does this indirectly, by creating a report object, notifying observers, and then queuing the report for sending. That report is later discarded when it realizes that there is no valid endpoint with that (null) name.

I filed https://github.com/w3c/reporting/issues/265 to officially support that pattern, which right now works by accident.

Reply:
OK, thanks that sounds good!